README.md

```diff
@@ -367,24 +367,56 @@ Either one of commands or closure needs to be specified.
 |    version     |    false   |     latest     |            version of the environment you want                 |
 |  checkout_scm  |    false   |      true      |    whether you want git code or not for performing commands    |
 |    commands    |    false   |      null      |             commands that you want to execute                  |
+|    envVar      |    false   |      []        |     Environment variables you want to set in the pod           |
```

Comment:
@rupalibehera technically this pram name should be `envvars` as it can accept more than one values.
Would you like to update te patch?